infra/testApi.mjs

```diff
@@ -3,177 +3,317 @@ import { strict as assert } from 'node:assert';  // use strict equality everywhe
 // Future TODO: before this API becomes set in stone/offered publicly, we should change the results of these methods to be just the output data rather than duplicating input values.
 // Reusable testing data
 const SAMPLE_SIZE = 8000
+const CHECK_CORS = true
 const FPCoreFormula = '(FPCore (x) (- (sqrt (+ x 1)) (sqrt x)))'
 const FPCoreFormula2 = '(FPCore (x) (- (sqrt (+ x 1))))'
 const FPCoreFormula3 = '(FPCore (x) (if (<= (- (sqrt (+ x 1.0)) (sqrt x)) 0.05) (* 0.5 (sqrt (/ 1.0 x))) (fma (fma (- 0.125) x 0.5) x (- 1.0 (sqrt x)))))'
 const eval_sample = [[[1], -1.4142135623730951]]
 
-// improve endpoint
-const improveResponse = await fetch(makeEndpoint(`/improve?formula=${encodeURIComponent(FPCoreFormula2)}`), { method: 'GET' })
-assert.equal(improveResponse.status, 200)
-let redirect = improveResponse.url.split("/")
-const jobID = redirect[3].split(".")[0]
-// This test is a little flaky as the character count of the response is not consistent.
-// const improveHTML = await improveResponse.text()
-// const improveHTMLexpectedCount = 25871
-// assert.equal(improveHTML.length, improveHTMLexpectedCount, `HTML response character count should be ${improveHTMLexpectedCount} unless HTML changes.`)
-
-// timeline
-const timelineRSP = await fetch(makeEndpoint(`/timeline/${jobID}`), { method: 'GET' })
-assert.equal(timelineRSP.status, 201)
-const timeline = await timelineRSP.json()
-assert.equal(timeline.length > 0, true)
+const det_44 = `(FPCore  (x y z t a b c i j k y0 y1 y2 y3 y4 y5)
```

Comment:
You don't need to copy this whole giant test case in! The reason this test crashes is just because it has a lot of variables. You could as well just write a one-line FPCore with 20 variables whose body is just `0`. (The test will also run faster.)